### ISSUE TITLE
[Bug]: config多卡下载报错

### ISSUE THREAD

**Issue by gongel** [PaddlePaddle/PaddleNLP#4275]
### 软件环境

```Markdown
- paddlepaddle: 
- paddlepaddle-gpu: develop
- paddlenlp: develop
```

### 重复问题

- [X] I have searched the existing issues

### 错误描述

```Markdown
config多卡下载报错
```

### 稳定复现步骤 & 代码

<img width="1109" alt="4ce8bba3197e225245bb465dfc44537e" src="https://user-images.githubusercontent.com/24390500/209895016-bc86ae93-fb46-4d80-b27c-eb0a30bb9bc4.png">

**Comment by sijunhe** [comment]
@gongel #4274 已合入，确认一下是否fix了

**Comment by gongel** [comment]
Fixed by #4274
